### ISSUE TITLE
API 1.1

### ISSUE THREAD

**Issue by moebiussurfing** [diederickh/ofxTwitterStream#3]
any way to make it work with the new tweeter API?

**Comment by roxlu** [comment]
@moebiussurfing yep, the difference with this version is that it uses http insteads of https. If you add https it will work.

**Comment by moebiussurfing** [comment]
thanks! I'll try

**Comment by jvelazqueztraut** [comment]
@moebiussurfing Did you have any luck on this?

**Comment by moebiussurfing** [comment]
@jvelazqueztraut some months ago... i finished using a Python library outside OF. (I linked both apps with OSC. I am not sure but maybe I used SQLite too.) Maybe you can check ofxJSON
